extensions/amp-truncate-text/1.0/amp-truncate-text.css

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-truncate-text {
+  padding: 0 !important;
+  border: none !important;
+}
+
+amp-truncate-text[i-amphtml-truncate-expanded] {
+  height: auto !important;
+}
+amp-truncate-text[i-amphtml-truncate-expanded] > i-amphtml-sizer {
+  display: none !important;
+}
+
+[i-amphtml-truncate-child-overflow] {
```

Comment:
I might be able to create a class with JSS and assign it to the root element, getting rid of this CSS file completely, but wasn't sure if that's the "right" way to go about it.

Reply:
As I mentioned in a different comment, these styles are "internal" so they're fine to move into JSS: the validator forbids authored CSS with `.i-ampthml-` and `!important`.

Reply:
Removal of CSS and shift to JSS-only is currently functional in my local branch (Preact mode, not Bento mode yet). Will push soon